vispy/glsl/build-spatial-filters.py

```diff
@@ -654,9 +655,37 @@ def weight(self, x):
 print("")
 print("const float kernel_bias  = %f;" % bias)
 print("const float kernel_scale = %f;" % scale)
+print("const float kernel_size = %f;" % n)
 print("uniform sampler2D u_kernel;")
 print("")
 
+code = 'float\n'
+code += 'unpack(vec4 rgba)\n'
+code += '{\n'
+code += '\t// return rgba.r;  // uncomment this for r32f debugging\n'
+code += '\trgba.rgba = rgba.abgr * 255;\n'
+code += '\tconst vec4 bits = vec4(1.0/(256.0*256.0*256.0), 1.0/(256.0*256.0), 1.0/256.0, 1.0);\n'
```

Comment:
you can move this outside the function to make it clear it's constant regardless of the input